src/util.rs

```diff
@@ -45,6 +46,7 @@ impl Default for Config {
             keep_list: false,
             strict: false,
             pascal_enum_values: false,
+            derive_more: false,
```

Comment:
I think the default should be that it's enabled. That probably means the flag needs to be negated, e.g `derive_less` perhaps

Reply:
`no_derive_more` ?

Reply:
I take that back, it should not be defaulted to on

Reply:
As this requires advanced dependency, it is breaking change for all svd2rust-generated crates.
So I don't think it is good idea to make it default.